rpcs3/Emu/Cell/PPUInterpreter.cpp

```diff
@@ -3529,7 +3576,9 @@ bool ppu_interpreter::FSQRTS(PPUThread& ppu, ppu_opcode_t op)
 
 bool ppu_interpreter::FRES(PPUThread& ppu, ppu_opcode_t op)
 {
-	ppu.FPR[op.frd] = f32(1.0 / ppu.FPR[op.frb]);
+	f32 value = f32(ppu.FPR[op.frb]);
+	_mm_store_ss(&value, _mm_rcp_ss(_mm_load_ss(&value)));
+	ppu.FPR[op.frd] = value;
```

Comment:
This will break if games rely on weird floating-point bullshit. And they probably do.